lib/broccoli/ember-app.js

```diff
@@ -409,14 +409,6 @@ class EmberApp {
             production: productionEmber,
           },
           'ember-testing.js': [emberTesting, { type: 'test' }],
-          'ember-resolver.js': [
-            `${this.bowerDirectory}/ember-resolver/dist/modules/ember-resolver.js`,
```

Comment:
Bower is not supported anymore. Also highly unlikely this even works in a modern Ember app.